apps/editing-toolkit/editing-toolkit-plugin/common/index.scss

```diff
@@ -2,4 +2,10 @@ body.hide-homepage-title {
 	.editor-post-title {
 		display: none;
 	}
+
+	// If the title is hidden in the block editor, leave enough space
+	// above the first block for the toolbar to show correctly.
+	.block-editor-block-list__layout.is-root-container > .wp-block:first-child {
```

Comment:
We can more selectively target what we need here with the `.hide-homepage-title` class [added here](https://github.com/Automattic/wp-calypso/blob/720b5dad4f5217adc9c1de3e2d706b56f6bb7cf9/apps/editing-toolkit/editing-toolkit-plugin/common/index.php#L86) when the title is hidden. This will prevent this style from being applied to themes that don't hide the title.

Reply:
This CSS rule should already be nested under `body.hide-homepage-title` — you can see it by revealing the first few lines of the file